testdata/ansible/memcached-operator/requirements.yml

```diff
@@ -1,10 +1,10 @@
 ---
 collections:
   - name: community.kubernetes
-    version: "1.2.1"
+    version: "2.0.1"
   - name: operator_sdk.util
     version: "0.4.0"
   - name: kubernetes.core
-    version: "2.2.0"
+    version: "2.3.1"
   - name: cloud.common
-    version: "2.1.0"
+    version: "2.1.1"
```

Comment:
@camilamacedo86 The version for `cloud.common` doesn't match with the one added to the docs

https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.22.0/#for-ansible-based-language-operators-upgrade-cloudcommon-from-220-to-221

Reply:
@camilamacedo86 I've created a PR to correct the version https://github.com/operator-framework/operator-sdk/pull/5911